src/program.js

```diff
@@ -503,6 +503,12 @@ Example: $0 --help run.
         demandOption: false,
         type: 'boolean',
       },
+      'binary-args': {
+        describe: 'Browser binary options. More at - ' +
+        'https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options',
```

Comment:
Nit, let's indent this line as the other "multi-line" `describe` values in this file.

Reply:
wdyt about adding eslint ignore for this line? URL is too long and linter doesn't like it.

Reply:
@denar90 How about omitting the link and give it a more generic description? e.g. something like the following:

```
'binary-args': {
  describe: "Additional command line options passed to the Browser binary',
  ...  
}
``` 

In my opinion, it may be a reasonable choice, e.g. as part of #1392 we may use the `--binary-args` also to pass additional CLI options to Chromium browsers.

Otherwise we may opt to include a shortened link instead (e.g. you can use https://mzl.la/2CEvSV5  which point to the above link).

Reply:
>How about omitting the link and give it a more generic description?

SGTM 👍